### ISSUE TITLE
CodeGit - Unfinished features?

### ISSUE THREAD

**Issue by mach7** [Atheos/Atheos#247]
Was looking for a way in the editor to switch between branches, or even make a new branch and wasn't able to find it. Started looking for a way to add it and discovered some commented out code located in codegit/templates/transfer.php lines 50 to 54. 

Is this a feature that you're currently working on?

**Comment by HLSiira** [comment]
Sort of, my work has pretty much taken over all of my free time this year so I'm not actively working on Atheos right now, hoping to get more time soon as it's still very important to me.

The Git section is my biggest priority as the CLI tool added some security mechanisms that made it harder to use with Atheos, and it's just a cumbersome area of the codebase. When I first made it, I didn't fully wrap my head around certain features, such as Branches, and it's limiting progress so it needs a lot of changes.

Does that answer your question?

**Comment by mach7** [comment]
Yes it does. Thanks for the update. I have been in the same position as you are now. But have lately found myself with a LOT more time on my hands. So maybe if I can dig into the code a little more I may be able to help you build this part out and contribute to the project.

**Comment by mach7** [comment]
I didn't mean to close this.
